docroot/.htaccess

```diff
@@ -69,6 +69,20 @@ AddEncoding gzip svgz
 <IfModule mod_rewrite.c>
   RewriteEngine on
 
+  # Forbid access to install and update files.
+  RewriteRule ^(core/install|update)\.php - [F]
+
+  # Forbid access to admin, node, and user paths on domains other than admin.foia.gov and local.dojfoia.gov.
+  RewriteCond %{ENV:AH_SITE_ENVIRONMENT} ^(prod|test|dev)$
```

Comment:
Is it possible to check if the variable is empty, rather than hardcoding environments? Just in case more environments are added down the road?